tests/ci/run_fips_tests.sh

```diff
@@ -4,37 +4,23 @@
 
 source tests/ci/common_posix_setup.sh
 
+echo "Testing AWS-LC in FIPS Release mode."
+fips_build_and_test -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1
+
+# The AL2 version of Clang does not have all of the required artifacts for address sanitizer, see P45594051
+if [[ "${AWSLC_NO_ASM_FIPS}" == "1" ]]; then
+  echo "Building with Clang and testing AWS-LC in FIPS RelWithDebInfo mode with address sanitizer."
+  fips_build_and_test -DASAN=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_SHARED_LIBS=1
+else
+  echo "Not building with Clang and testing AWS-LC in FIPS RelWithDebInfo mode."
```

Comment:
What does this echo mean?

Reply:
The current CMake address sanitizer config only works with Clang, so in our CI for the GCC targets it doesn't test with ASAN.